akka-docs/src/test/java/jdocs/actor/typed/PrintActor.java

```diff
@@ -10,6 +10,10 @@
 // #print-actor
 class PrintActor extends AbstractBehavior<Integer> {
 
+  public static Behavior<Integer> create() {
+    return Behaviors.setup(notUsed -> new PrintActor());
```

Comment:
In this kind of place I'd rather call it `context` even if it isn't used.

Reply:
okidok, I agree